### PR TITLE
Wordle

### PR DESCRIPTION
**Pull request by aadityathukral**
Pull branch after I messed up Wordle 💀 (sorry Justin). 

Added some stylistic changes to Wordle, will update further.

### CODE REVIEW

File: public/uploads/wordle/wordguesser.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ async function init() {
   setLoading(isLoading);
 
   function addLetter(letter) {
+    console.log(letter);
```
Comment (mjh316) [inline public/uploads/wordle/wordguesser.js]: What's this for?

**Review by mjh316 — CHANGES_REQUESTED** [review]

File: public/uploads/wordle/wordguesser.js (hunk 1 above)
Comment (aadityathukral) [reply public/uploads/wordle/wordguesser.js]: Oh shit, that was for debugging purposes, I'll get rid of it the next time I commit some more changes

**Review by mjh316 — APPROVED** [review]
LGTM!

File: public/uploads/wordle/wordguesser.js (hunk 1 above)
Comment (mjh316) [reply public/uploads/wordle/wordguesser.js]: Alright, I'll go ahead and comment it out for now

**Comment by mjh316** [comment]
@don-mo , what do you think? (meant to tag above but forgot)

**Review by don-mo — CHANGES_REQUESTED** [review]
# Nice! 
### Maybe these changes as well
 - display: none; for the title tag
 
 - doesn't have to be for right now but creating an animation would be dope haaha
wordle_example

**Comment by mjh316** [comment]
@aadityathukral

**Comment by aadityathukral** [comment]
Hey @mjh316, Siddharth wanted to get up to date with the changes I've made so far as well, is there a way to do that or would this branch need to be merged first?

**Comment by don-mo** [comment]
@aadityathukral Ok, to let @sidlak-c137 get caught up with the work that's been created this is good!! Sorry, codeacademy told me that this is how GitHub collaboration works, but I'm still getting used to it :))

**Review by don-mo — COMMENTED** [review]
# Merging 👍 
For @sidlak-c137 to get caught up!!

**Review by don-mo — APPROVED** [review]

**Pull request merged**